### PR TITLE
camel-zipkin - ZipkinServerResponseAdapter NPE fix

### PR DESCRIPTION
**Pull request by p-8-z**
Fix Nullable Throwable#getMessage NPE on SpanCustomizer#tag method
https://issues.apache.org/jira/browse/CAMEL-16715

### CODE REVIEW

File: components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinServerResponseAdapter.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ void onResponse(Exchange exchange, SpanCustomizer span) {
         span.tag("camel.server.exchange.id", id);
         span.tag("camel.server.exchange.pattern", mep);
 
-        if (exchange.getException() != null) {
+        if (exchange.getException() != null && exchange.getException().getMessage() != null) {
```
Comment (davsclaus) [inline components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinServerResponseAdapter.java:48]: There is still a failure, so you should instead if the message is null, create a empty message, or use the exception classname as the message.

**Review by davsclaus — CHANGES_REQUESTED** [review]

File: components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinServerResponseAdapter.java (hunk 1 above)
Comment (p-8-z) [reply components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinServerResponseAdapter.java:48]: So I'll roll back that if statement and if _message_ is null (or blank???) I will change it to exception classname, is that right?

Comment (davsclaus) [reply components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinServerResponseAdapter.java:48]: Yes correct, then we always ensure that if an exception occurred then zipkin gets that as a failure event

Comment (p-8-z) [reply components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinServerResponseAdapter.java]: Change request done.

File: components/camel-zipkin/pom.xml [hunk 2]
```diff
@@ -45,6 +45,12 @@
             <artifactId>camel-support</artifactId>
         </dependency>
 
+        <dependency>
```
Comment (davsclaus) [inline components/camel-zipkin/pom.xml:48]: Please do not add a new dependency just for checking if a string is empty/black.

You can use Camel's own 

org/apache/camel/util/ObjectHelper.isEmpty

**Review by davsclaus — CHANGES_REQUESTED** [review]

File: components/camel-zipkin/pom.xml (hunk 2 above)
Comment (p-8-z) [reply components/camel-zipkin/pom.xml]: Change request done.

**Pull request merged**